react-native.config.js

```diff
@@ -19,11 +19,6 @@ module.exports = {
 			platforms: {
 				ios: null
 			}
-		},
-		'@notifee/react-native': {
-			platforms: {
-				ios: null
-			}
```

Comment:
This configuration was set by @GleidsonDaniel in this PR: https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/5286

Afterwards, I checked all the places where he worked, and he consistently verified if it is an Android device. Therefore, we can link Notifee normally for iOS since it will not affect his changes.

Places that he used the notifee:
https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/274b88701f49985525687e95381d6942ae62c7b4/index.js#L23-L25

https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/274b88701f49985525687e95381d6942ae62c7b4/app/lib/notifications/videoConf/getInitialNotification.ts#L6-L7